db/schema.rb

```diff
@@ -11,304 +10,294 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160930190451) do
+ActiveRecord::Schema.define(version: 20161005185412) do
```

Comment:
schema.rb gets recreated every time you run `rake db:migrate`... based on the state of your own database. But in tests and production, the schema is what gets loaded to control behavior.

In this case, the only things that should change are this line defining the prefix of the latest required migration...